app/models/categorie.js

```diff
@@ -0,0 +1,8 @@
+import Model from 'ember-data/model';
+
+export default Model.extend({
+	name: attr('string'),
+	ID: attr('number')'
+	userID: belongsTo('user')
```

Comment:
Same here - it's better to follow the convention and name this field `userId` - but of course that's only a cosmetic issue.